.github/workflows/upgrade-nodes-on-test-envs.yml

```diff
@@ -93,6 +93,14 @@ jobs:
                 nohup sh -c "docker logs -f juno_$network > /var/log/juno_logs_$network.log 2>&1" &>/dev/null &
             done
 
+    - name: Trigger starknet-rs tests
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: starknet-rs-tests.yml
+        token: ${{ secrets.REPOSITORY_DISPATCH_TOKEN }}
+        ref: ${{ github.ref }}
+        inputs: '{"NODE_URL": "http://${{ env.HOST }}:6065"}'
+
```

Comment:
I think we should put some delay here, for node to boot up, catch up to the network and download the pending block as well.

Reply:
Not sure if it's needed. Before the tests run, the workflow already spends time starting a new agent, checking out the `starknet-rs` repo, and setting up Rust which includes downloading and compiling. This should add enough of a delay on its own. I'd rather not add a fixed delay if we don't need to.